src/sysen2/name.558

```diff
@@ -264,6 +264,17 @@ $$ANALYZ==1
 .INSRT SYSTEM;CHSDEF
 .INSRT SYSENG;NETWRK
 
+;; default off. Compile with /T to input DODNS==:1
+ifndef DODNS,DODNS==:0
```

Comment:
Minor tiny silly comment, but I'd say DNSP would be more idiomatic.

Reply:
Does it make sense to require /T, or would .TTYMAC be better? I'm assuming most people would not want the DNS complication (but instead the H3TEXT complication), but I'm not sure.